tools/print_test_stats.py

```diff
@@ -599,16 +599,13 @@ def append(self, test_case: TestCase) -> None:
         self.skipped_count += 1 if test_case.skipped else 0
         self.errored_count += 1 if test_case.errored else 0
 
-    def replace(self, test_case: TestCase) -> float:
+    def update(self, test_case: TestCase) -> None:
         name = test_case.name
         assert name in self.test_cases, f'Error: attempting to replace nonexistent test case {name}'
-        old_time = self.test_cases[name].time
-        # We don't replace anything if the old test case was not shorter.
-        if old_time >= test_case.time:
-            return 0.0
-        self.total_time = self.total_time + test_case.time - old_time
-        self.test_cases[name] = test_case
-        return test_case.time - old_time
+        self.test_cases[name].time += test_case.time
```

Comment:
So is the design decision to take the sum of the times instead of the max? What about tests that were run in parallel? (This was the preliminary reason why we did max instead of sum.)

Reply:
do you have an example of test that ran in parallel? (from what i can see all those distributed tests are ran sequentially)

Reply:
Ah, I think I previously misunderstood how the tests were spawned. Since `test_distributed_spawn` seems to be just spawn sequentially in `run_test.py` multiple times, summing is probably better. The downside is that there are other configurations where we might be better off taking the max (like test_cpp_extensions), but those seem like smaller tests in general.